std/getopt.d

```diff
@@ -488,9 +483,7 @@ enum config {
 
 /** The result of the $(D getopt) function.
 
-The $(D GetoptResult) contains two members. The first member is a boolean with
-the name $(D helpWanted). The second member is an array of $(D Option). The
-array is accessable by the name $(D options).
+$(D helpWanted) is set if the option `--help` or `-h` was passed to the option parser.
```

Comment:
I'm not sure if less informatik is good here, as people may want to implement their own GetoptResult writer.